src/components/icons/InfoIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { createIcon } from '@chakra-ui/react';
+
+export const InfoIcon = createIcon({
+  displayName: 'Info',
+  viewBox: '0 0 24 24',
+  path: (
+    <>
+      <title>Info icon</title>
+      <path
+        d="M20 10C20.0018 8.59528 19.6337 7.21485 18.9327 5.99754C18.2317 4.78023 17.2225 3.76896 16.0066 3.06543C14.7908 2.36191 13.4111 1.99093 12.0064 1.98981C10.6017 1.98869 9.2214 2.35747 8.00442 3.05905C6.78745 3.76064 5.77666 4.7703 5.07371 5.98649C4.37077 7.20268 4.00045 8.58253 4 9.98725C3.99955 11.392 4.36899 12.7721 5.07116 13.9887C5.77332 15.2053 6.78347 16.2156 7.99999 16.918V22H16V16.918C17.2148 16.2169 18.2238 15.2086 18.9259 13.9944C19.628 12.7801 19.9984 11.4026 20 10ZM14 20H9.99999V17.737C11.3102 18.0881 12.6898 18.0881 14 17.737V20ZM13 15.91V11H15V9.00001H8.99999V11H11V15.91C9.5166 15.6593 8.18152 14.8604 7.25935 13.6717C6.33718 12.4831 5.8952 10.9913 6.02101 9.49217C6.14682 7.993 6.83124 6.5958 7.93862 5.57745C9.04601 4.5591 10.4956 3.99393 12 3.99393C13.5044 3.99393 14.954 4.5591 16.0614 5.57745C17.1688 6.5958 17.8532 7.993 17.979 9.49217C18.1048 10.9913 17.6628 12.4831 16.7406 13.6717C15.8185 14.8604 14.4834 15.6593 13 15.91Z"
```

Comment:
Can we optimize this? Looks like it's 1:1 from the figma export? (They tend to be very large you can use something like svgo to have a smaller component)

Reply:
Good point, done